lib/systems/default.nix

```
@@ -76,6 +76,7 @@ rec {
          # uname -r
          release = null;
       };
+      isStatic = false;
```

It is good to set a default here, but let's make it a bit more nuanced, based on https://github.com/NixOS/nixpkgs/blob/ab247348cbce60cf2c6b8a4a3ce70e25528265c0/pkgs/stdenv/cross/default.nix#L41
```suggestion
      isStatic = final.isWasm || final.isRedox;
```